docs/book/driver-deployment/deploying_csi_with_zones.md

```diff
@@ -0,0 +1,314 @@
+<!-- markdownlint-disable MD033 -->
+# vSphere CSI Driver - Setup Topology (Zoning)
+
+- [Set Up Zones in the vSphere CNS Environment](#set_up_zones_in_vspehre)
+- [Enable Zones for the vSphere CPI and CSI Driver](#enable_zones_for_vsphere_cpi_and_csi)
+- [Deploy Workloads Using Zones](#deploy_workload_using_zones)
+
+When you deploy CPI and CSI in a vSphere environment that includes multiple data centers or host clusters, you can use zoning.
+
+Zoning enables orchestration systems, like Kubernetes, to integrate with vSphere storage resources that are not equally available to all nodes. As a result, the orchestration system can make intelligent decisions when dynamically provisioning volumes, and avoid situations such as those where a pod cannot start because the storage resource it needs is not accessible.
+
+## Set Up Zones in the vSphere CNS Environment <a id="set_up_zones_in_vspehre"></a>
+
+Depending on your vSphere storage environment, you can use different deployment scenarios for zones. For example, you can have zones per host cluster, per data center, or have a combination of both.
+
+In the following example, the vCenter Server environment includes three clusters with node VMs located on all three clusters.
+
+![REGION_AND_ZONES](https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/docs/images/REGION_AND_ZONES.png)
+
+The sample workflow creates zones per cluster and per data center.
+
+Procedure
```

Comment:
Can you highlight this.
Procedure ==> #### Procedure

Reply:
done